files/image_config/updategraph/updategraph

```diff
@@ -1,12 +1,80 @@
 #!/bin/bash
 
+reload_minigraph()
+{
+    echo "Reloading minigraph..."
+    if [ ! -f /etc/sonic/init_cfg.json ]; then
+        echo "{}" > /etc/sonic/init_cfg.json
+    fi
+    redis-cli -n 4 FLUSHDB
```

Comment:
>4 [](start = 17, length = 1)

can you assign config_dbnum to 4 and use it throughout the script?

Reply:
Done.